flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -506,6 +506,30 @@ public void testUseCatalogAndUseDatabase() throws Exception {
 		}
 	}
 
+	@Test
+	public void testUseNonExistingDatabase() throws Exception {
+		final Executor executor = createDefaultExecutor(clusterClient);
+		final SessionContext session = new SessionContext("test-session", new Environment());
+
+		try {
+			executor.useDatabase(session, "nonexistingdb");
+		} catch (SqlExecutionException e) {
```

Comment:
shall we just assert the exception is expected instead of try-catch-ignore?

Reply:
It's following the pattern in `testValidateSession`. But I guess it's also fine to use `ExpectedException`.